flytekit/core/array_node_map_task.py

```diff
@@ -77,8 +77,9 @@ def __init__(
             f = actual_task.lhs
         else:
             _, mod, f, _ = tracker.extract_task_module(cast(PythonFunctionTask, actual_task).task_function)
+        sorted_bounded_inputs = ",".join(sorted(self._bound_inputs))
```

Comment:
note: could also just turn self._bound_inputs into a sorted list as opposed to set - don't feel too strongly either way